docs/documentation/platform/sso/overview.mdx

```diff
@@ -39,18 +39,30 @@ If your required identity provider is not shown in the list above, please reach
   For enhanced security, Infisical enforces PKCE (Proof Key for Code Exchange) with the OAuth 2.0-based SSO providers and OIDC. This provides additional protection against authorization code interception attacks and strengthens your authentication flow security.
 </Info>
 
+## Admin Login Portal
```

Comment:
I think we should title this SSO Break Glass. Folks might get confused with Admin Portal and think it's for other functions.  This will keep bypass in the spotlight